readme.md

```diff
@@ -303,6 +303,23 @@ typically yield `SerializableError`s with `exceptionClass=errorCode=<exception c
 user directly. Services should prefer to throw `ServiceException`s instead of the above, since they are easier to
 consume for clients and support transmitting exception parameters in a safe way.
 
+##### `RemoteException` vs `ServiceException` vs `SerializableError` vs `ErrorType`
+
+ - `ErrorType` is a record type, meant to be used as 'compile time constants' - essentially used by services to define the 'enum' of their service exceptions
+ - `SerializableError` is a record type, containing things like error code, error instance id, params - used as the wire format
```

Comment:
`SerializableError` defines the wire format for serializing `ServiceException`s in HTTP response bodies and contains the error code, error instance id, and application-defined paramaters